### DISCUSSION TITLE
Get a JSON with all variants for a style

### DISCUSSION THREAD

**Discussion by Bruno-Lafayette** [dicebear/dicebear#265] [Q&A] [answered]
I was reading the documentation and couldn't find anywhere a way to, when using a style, get all options for a hair variant or something similar. It would be better for me in order to avoid hardcoding all variants.

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @Bruno-Lafayette 

Excellent question! I added the answer to the documentation right away. It can be found here:
https://dicebear.com/guides/access-all-available-options
